canonical-web-and-design/ubuntu.com
# Tutorials topic filter

(#6284)[https://github.com/canonical-web-and-design/ubuntu.com/pull/6284] must be merged before this can be merged

## Done
Add the topics filter to the tutorials homepage

## QA
- Go to the tutorials homepage: `/tutorials`
- Change the topics filter and see that only tutorials that match that topic are visible

## Issue / Card

Fixes #6211
